sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -67,6 +67,13 @@ Licensed under the MIT License.
       <artifactId>azure-core</artifactId>
       <version>1.17.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.azure/azure-core-tracing-opentelemetry -->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-tracing-opentelemetry</artifactId>
+      <version>1.0.0-beta.11</version>
```

Comment:
This is temporary change due to bug in event api https://github.com/Azure/azure-sdk-for-java/issues/22115 . Will revert before merging the PR

Reply:
removed temporary change